internal-api/src/main/kotlin/systems/ajax/malov/internalapi/KafkaTopic.kt

```diff
@@ -0,0 +1,13 @@
+package systems.ajax.malov.internalapi
+
+object KafkaTopic {
+    private const val REQUEST_PREFIX = "ajax.systems.stockanalyzer.output.pub"
+    const val STOCK_PRICE_CONSUMER_GROUP = "stockPriceConsumerGroup"
```

Comment:
consumer group should not be placed in internal API. Kafka topic is common and shared information in all your 'cloud'. consumer group on the other hand is specific for each consumer, so it should be located together with consumer